CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Using the following categories, list your changes in this order:
 ### Added
 
 -   Support Python 3.13.
+-   Django `settings.py:SERVESTATIC_PRESERVE_QUERY_STRING_ON_REDIRECT` will forward the request query string when issuing redirects.
```

Comment:
Given that there is no detriment to preserving query strings, I don't believe we should make a setting out of this.

It feels like the behavior of preserving query strings should always be on.

Reply:
Removed the feature flag in fdfbf1d